mutmut/__main__.py

```diff
@@ -426,7 +459,12 @@ def split_paths(paths):
     else:
         return compute_exit_code(progress, ci=ci)
     finally:
-        print()  # make sure we end the output with a newline
+        # want newline at the end of the progress output PLUS extra blank line before the following hint text
+        print()
+        print()
+        print("To show the results: mutmut results")
+        print()
+        print("To generate HTML report: mutmut html")
```

Comment:
Now removed. I had thought this could be helpful for newcomers, just like `mutmut results` shows how to show and apply a surviving mutant. Maybe another, shorter hint could be ideal, even if it just says to do `mutmut --help` for pointers on what to do next.